ee/clickhouse/queries/funnels/base.py

```diff
@@ -248,7 +248,7 @@ def _get_breakdown_select_prop(self) -> str:
             self.params.update({"breakdown": self._filter.breakdown})
             if self._filter.breakdown_type == "person":
                 return f", JSONExtractRaw(person_props, %(breakdown)s) as prop"
-            elif self._filter.breakdown_type == "event":
+            elif self._filter.breakdown_type == "event" or not self._filter.breakdown_type:
```

Comment:
I guess need the same for breakdown_conditions ?

Reply:
Hmmm, at this point, perhaps better to add the default in `exec_query` ? Like we do for dates?

Reply:
done